app/webpacker/controllers/searchbox_controller.js

```diff
@@ -77,4 +94,40 @@ export default class extends Controller {
     if (result)
       return result.html
   }
+
+  get redactedQuery() {
+    return (new Redacter(this.searchQuery)).redacted
+  }
+
+  get google_analytics_id() {
+    const ga_id = document.body.getAttribute('data-analytics-ga-id')
+
+    if (ga_id && ga_id.trim() != "")
+      return ga_id
+  }
+
+  scheduleSubmissionToAnalytics() {
+    this.clearAnalyticsSubmitter() ;
+
+    this.analyticsSubmitter = window.setTimeout(this.sendToAnalytics.bind(this), 2000)
+  }
+
+  sendToAnalytics() {
+    this.clearAnalyticsSubmitter() ;
+
+    if (window.ga && this.google_analytics_id) {
```

Comment:
Is there a reason we're using google analytics directly here instead of tag manager? I'm _fairly_ sure you can create and write page view events to `dataLayer` directly and GTM will pick it up instead of going directly to GA.

Reply:
Yes, in theory - that was my thought but I was never able to get it to work. The way GA operates is you have watch for things happening, and hook those observers up to actions. So in theory you can create an event in GTag, than get that event to send the page view.

I couldn't get that to work and just sending straight to GA was _a lot_ easier - it should be lower effort if we need to send other stuff through to GA in the future. Tbh I feel GTag just adds to the complexity rather than reducing it.